entry_types/scrolled/package/src/frontend/useSectionChangeEventEmitter.js

```diff
@@ -0,0 +1,11 @@
+import {events} from 'pageflow/frontend';
+
+export const useSectionChangeEventEmitter = (entryStructure) => (index) => {
+  const getSectionChapterTitle = (sectionId, delimiter = '') => entryStructure.filter(
+    chapter => chapter.sections.some(section => section.id === sectionId)
+  )[0]?.title + delimiter;
+
+  events.trigger('page:change', {configuration: {
+    title: getSectionChapterTitle(index, ', ') + 'Section ' + index
+  }, index: index});
```

Comment:
Section id and index are two independent concepts. Try to extend the test to make it check the title of the chapter is looked up correctly and passed in the event.